tests/test_organization.py

```diff
@@ -35,18 +36,20 @@
 )
 from tests import TEST_USER_ID, TestCase
 
+
 class TestOrganization(TestCase):
     """Organization model tests"""
 
     def test_from_fhir(self):
         with open(os.path.join(
-            os.path.dirname(__file__),
-            'organization-example-f001-burgers.json'), 'r') as fhir_data:
+                os.path.dirname(__file__),
+                'organization-example-f001-burgers.json'), 'r') as fhir_data:
```

Comment:
Please put `) as fhir_data:` on its own un-indented line (aligned with the `with`) to make it obvious its not part of the `with` block